### ISSUE TITLE
Rock Band 3 (BLUS30463) can't switch between Quickplay/Practice without restarting

### ISSUE THREAD

**Issue by SCOTT0852** [RPCS3/rpcs3#12840]
### Quick summary

After playing some songs in Quickplay, attempting to start a song in Practice will crash the game. The bug sometimes won't happen if you've only played one song in Quickplay that session.

### Details

On console, you can play any order of songs in either mode and the game won't crash. In the provided log I was able to successfully enter Practice after playing one song in Quickplay, but after playing a few more opening Practice crashed the game. For what it's worth (which probably isn't much, admittedly) the issue does not happen in Xenia or Dolphin with their versions of the game.

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9817510/RPCS3.log.gz)

### System configuration

Windows 11 22H2 Insider build 22623.746
AMD Ryzen 5 3600
NVidia GTX 1660 super, driver 522.25

**Comment by FlexBy420** [comment]
Is this still an issue?

**Comment by shny-scott** [comment]
Yep, just reproduced on 0.0.33-017046. For what it's worth, in the time since this issue was opened the issue was fixed in the Rock Band 3 Deluxe mod by adding a two second delay to loading screens. It's been theorized to be some kind of race condition that happens even on console (very rarely) after extended play sessions. Only took two songs before practice mode to trigger it today, so I don't think it's quite hardware accurate.

**Comment by digant73** [comment]
@shny-scott is that still an issue?

**Comment by shny-scott** [comment]
Yes, I just reproduced it on 0.0.38-18263-3c401e7b.
